src/builds/node/launch-build.ts

```diff
@@ -0,0 +1,80 @@
+import { IBuildOutputParsedEntry } from './build-parser';
+import { GccBuildParser } from './gcc-build-parser';
+import * as Events from "events"
+import * as child from 'child_process';
+
+
+export interface LaunchBuildinfo {
+    /** the directory where to execute the build */
+    cwd: string
+    /** build executable or script to call */
+    buildCommand: string,
```

Comment:
Command infers to me that I may be able to send arguments directly there.. should it be more like buildExecutablePath or something ?